### PR TITLE
Bug fix for https://bugs.launchpad.net/or/+bug/1983511 When an AI train disappears at end of run, containers remain visible

### PR DESCRIPTION
**Pull request by Csantucci**

### CODE REVIEW

File: Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/FreightAnimations.cs [hunk 1]
```diff
@@ -863,6 +863,22 @@ public bool CheckForMerge(int i)
             }
             return false;
         }
+
+        /// <summary>
+        /// Removes discrete freight animations and containers when AI train is deleted
+        /// </summary>
+
+        public void RemoveDiscreteFreightAnimations(MSTSWagon wagon)
```
Comment (Sharpe49) [inline Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/FreightAnimations.cs:871]: Shouldn't the function by called RemoveContainers?
Can't you use the Wagon member which is already present in the class?

**Review by Sharpe49 — CHANGES_REQUESTED** [review]

File: Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/FreightAnimations.cs (hunk 1 above)
Comment (Csantucci) [reply Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/FreightAnimations.cs]: OK, I have updated the code.

**Review by Sharpe49 — APPROVED** [review]

**Comment by Csantucci** [comment]
Cédric, thank you.

**Comment by Sharpe49** [comment]
You should probably try to merge this branch in release/1.5

**Pull request merged**